wasmer/instance.go

```diff
@@ -220,6 +222,13 @@ func NewInstanceFromCompiledCodeWithOptions(
 		c_instance_context := cWasmerInstanceContextGet(c_instance)
 		instance.InstanceCtx = IntoInstanceContextDirect(c_instance_context)
 	}
+
+	hackMap := make(map[unsafe.Pointer]interface{})
```

Comment:
why is the hackMap needed?
aren't just lines 206 and 230 relevant?

Reply:
testing now - as without it is fails in some scenarios.